src/ggrc/models/revision.py

```diff
@@ -742,13 +752,29 @@ def content(self):
     # remove attribute_object_id not used by FE anymore
     for item in populated_content["custom_attribute_values"]:
       item.pop("attribute_object_id", None)
+
+    self.set_cached_content(self.id, populated_content)
+
     return populated_content
 
   @content.setter
   def content(self, value):
     """ Setter for content property."""
     self._content = value
 
+  def get_cached_content(self):
+    """Return cached content value if it is"""
+    if hasattr(flask.g, 'rev_content') and flask.g.rev_content.get(self.id):
```

Comment:
The following condition will be more correct. With current condition the `{}` value in cache content still will result in content recalculation which might not be valid. Only `None` value should be considered as ***"content is not cached"***.
```suggestion
    if hasattr(flask.g, 'rev_content') and self.id in flask.g.rev_content:
```